qlib/backtest/__init__.py

```diff
@@ -199,12 +217,29 @@ def get_strategy_executor(
     )
 
     exchange_kwargs = copy.copy(exchange_kwargs)
+    # derive freq from executor config if not explicitly provided
+    if "freq" not in exchange_kwargs:
+        try:
+            if isinstance(executor, dict):
+                tps = executor.get("kwargs", {}).get("time_per_step")
+                if isinstance(tps, str) and tps:
+                    exchange_kwargs["freq"] = tps
+        except Exception:
+            pass
+
     if "start_time" not in exchange_kwargs:
         exchange_kwargs["start_time"] = start_time
     if "end_time" not in exchange_kwargs:
         exchange_kwargs["end_time"] = end_time
     trade_exchange = get_exchange(**exchange_kwargs)
 
+    # align account/report frequency with exchange frequency to avoid inconsistent benchmark frequency
```

Comment:
Why do we have to reset it, 
It will be reset when using it with the according granularity.
<img width="1134" height="429" alt="image" src="https://github.com/user-attachments/assets/b4e30534-01d5-4af7-9835-c1902f0e2150" />